conf/app.transportEquipment.index.routes

```diff
@@ -1,15 +1,15 @@
 ->         /                                                                                app.transportEquipment.index.seals.Routes
 
-GET        /:arrivalId/transport-equipment/:equipmentIndex/items                                                                  controllers.transportEquipment.index.GoodsReferenceController.onPageLoad(arrivalId: ArrivalId, equipmentIndex:Index, mode: Mode = NormalMode)
-POST       /:arrivalId/transport-equipment/:equipmentIndex/items                                                                  controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index, mode: Mode = NormalMode)
-GET        /:arrivalId/transport-equipment/:equipmentIndex/change-items                                                           controllers.transportEquipment.index.GoodsReferenceController.onPageLoad(arrivalId: ArrivalId, equipmentIndex:Index, mode: Mode = CheckMode)
-POST       /:arrivalId/transport-equipment/:equipmentIndex/change-items                                                           controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index, mode: Mode = CheckMode)
+GET        /:arrivalId/transport-equipment/:equipmentIndex/:goodsReferenceIndex/items                                             controllers.transportEquipment.index.GoodsReferenceController.onPageLoad(arrivalId: ArrivalId, equipmentIndex:Index, goodsReferenceIndex: Index, mode: Mode = NormalMode)
```

Comment:
I think it would be better for us to put the `:goodsReferenceIndex` after items. Because then it'll read more naturally such as transport equipment -> equipment number -> items -> item number